### PR TITLE
Add kafka for Python

### PR DESCRIPTION
**Pull request by Crell**

### CODE REVIEW

**Comment by chadwcarlson** [comment]
Missing Kafka from the "What relationships look like" front section?

File: python/examples/kafka.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        
+        # Producer
+        producer = KafkaProducer(bootstrap_servers=[kafka_server],
+        value_serializer=lambda x: dumps(x).encode('utf-8'))
+        for e in range(10):
+            data = {'number' : e}
+            producer.send('numtest', value=data)
+        
+        # Consumer
+        consumer = KafkaConsumer(
```
Comment (chadwcarlson) [inline python/examples/kafka.py]: `consumer = KafkaConsumer(
  bootstrap_servers=[kafka_server],
  auto_offset_reset='earliest')`

**Comment by Crell** [comment]
Good catch on the relationships section.  I'll be able to add that once I've written the PHP version, since the relationships section is PHP-powered.

**Comment by Crell** [comment]
My previous comment about needing the PHP version is completely wrong, so ignore that entirely.

**Pull request merged**